sync.js

```diff
@@ -97,6 +97,14 @@ class SyncState {
     }
   }
 
+  activePeers () {
+    return this.peers().filter(this._isactive)
+  }
+
+  _isactive (peer) {
```

Comment:
This and `_isclosed` don't need to be methods (they don't reference `this`, and aren't public methods) so they could be standalone functions. But that's just a stylistic thing.

Reply:
Well, the `peer` object is encapsulated by the SyncState class, so these these methods should be in the SyncState (where the peer object is created and managed) to maintain that. Ideally, the `peer` object would be it's own class and these methods would live there.